app/config/settings.py

```diff
@@ -1220,6 +1224,10 @@ def sentry_before_send(event, hint):
         "task": "grandchallenge.emails.tasks.cleanup_sent_raw_emails",
         "schedule": crontab(hour=6, minute=0),
     },
+    "cancel_external_evaluations_past_timeout": {
+        "task": "grandchallenge.evaluation.tasks.cancel_external_evaluations_past_timeout",
+        "schedule": crontab(hour=6, minute=15),
```

Comment:
Is once a day enough? This could potentially leave evaluations running for 47:59:59 hours.